lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -185,22 +181,14 @@ def va_profile_opt_in_out_lambda_handler(event: dict, context, worker_id=None) -
     """
 
     logger.debug("POST event: %s", event)
-    global requested_certificates, va_profile_public_cert, va_root_pem
-
-    if not requested_certificates:
-        va_profile_public_cert, va_root_pem = get_certificates_from_ssm()
-
-        if va_profile_public_cert is None:
-            sys.exit("Cannot verify JWT signatures on POST requests from VA Profile.")
-
-        requested_certificates = True
+    global va_profile_public_cert
 
     # Authenticate the request from VA Profile.
     headers = event.get("headers", {})
     if not jwt_is_valid(headers.get("Authorization", headers.get("authorization", '')), va_profile_public_cert):
         return { "statusCode": 401 }
 
-    post_body = event["body"]
+    post_body = event.get("body")
```

Comment:
If the event doesn't contain "body", post_body is assigned None, and the handler exits on line 200.

Reply:
Good catch!